### ISSUE TITLE
java.lang.RuntimeException: java.lang.RuntimeException: Could not get cyclomatic complexity

### ISSUE THREAD

**Issue by fabio-franco** [IBM/codenet-minerva-code-analyzer#76]
A runtime exception is got when running code analyzer on erww-lite project:

```
[36m2024-11-15T12:43:41.183108700[0m[32m	[DONE]	[0mFinished construction of call graph. Took 122.0 seconds.
[36m2024-11-15T12:43:41.464701800[0m[31m	[ERROR]	[0mNull pointer exception in getCyclomaticComplexity
java.lang.RuntimeException: java.lang.RuntimeException: Could not get cyclomatic complexity.
	at com.ibm.cldk.CodeAnalyzer.run(CodeAnalyzer.java:105)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1767)
	at picocli.CommandLine.access$900(CommandLine.java:145)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2139)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2106)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:1973)
	at picocli.CommandLine.execute(CommandLine.java:1902)
	at com.ibm.cldk.CodeAnalyzer.main(CodeAnalyzer.java:94)
Caused by: java.lang.RuntimeException: Could not get cyclomatic complexity.
	at com.ibm.cldk.utils.AnalysisUtils.getCyclomaticComplexity(AnalysisUtils.java:97)
	at com.ibm.cldk.SystemDependencyGraph.lambda$construct$11(SystemDependencyGraph.java:249)
	at java.base/java.lang.Iterable.forEach(Iterable.java:75)
	at com.ibm.cldk.SystemDependencyGraph.construct(SystemDependencyGraph.java:246)
	at com.ibm.cldk.CodeAnalyzer.analyze(CodeAnalyzer.java:177)
	at com.ibm.cldk.CodeAnalyzer.run(CodeAnalyzer.java:103)
	... 7 more
```

The code analyzer command used (Readme.txt), the project files, and the full output of the issue file are available in this folder: https://ibm.box.com/s/zule6rqtra8cok8g3w4pzrn0saae9s5m

**Comment by rahlk** [comment]
@fabio-franco We are unable to build this application. Can you please recommend how to build this application?

**Comment by fabio-franco** [comment]
This seems fixed on code analyzer 1.0.6. Thank you!
